### ISSUE TITLE
Adapt the contribute category when there's no way to contribute

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#5041]
https://opencollective.com/indeed
![Screen Shot 2021-12-14 at 3 14 55 PM](https://user-images.githubusercontent.com/5498878/145920730-ba8bc286-5ae8-480e-9c5c-8c3bc8d20dcb.png)

The whole top half of the page is dominated by a contribute section with nothing in it. I don't think it's a good use of space. If a section has nothing in it, we shouldn't show it.

**Comment by Betree** [comment]
Technically, it's because the "Contributors" sections are part of the contribute category. I think it's the right move that we keep surfacing them, even though there's no way to contribute.

I however see a few issues there:
- The wording is not adapted. I would rephrase `Contribute` to `Contributions` when there's no way to contribute
- Same, the "Become a financial contributor" subtitle should only be displayed if contributions are allowed
- "Top financial contributors" appears to be empty. We should not display the title if there's no one to show in there.

**Comment by Betree** [comment]
This seems to also happen when the "Contribute" category is completely empty https://opencollective.com/akalaka/events/social-sibling-caregivers-giving-back-35cc18ed

![image](https://user-images.githubusercontent.com/1556356/147446341-daa9cf1d-dfa4-44a6-81fd-c63a267f2f37.png)
